Modules/xSharePoint/DSCResources/MSFT_xSPAppCatalog/MSFT_xSPAppCatalog.psm1

```diff
@@ -0,0 +1,79 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)] [System.String] $AppCatalogUrl ,
```

Comment:
Do we need app catalog URL and web app url here? Can't we just look up the web app that the site collection is in? Note that the update powershell method we use only takes the site in, not the web app URL as well.